src/routes/generalRoute.ts

```diff
@@ -0,0 +1,7 @@
+import express from "express";
+
+import { getResultsBySearch } from "../controllers/generalController";
+
+export default (router: express.Router) => {
+  router.get("/search", getResultsBySearch);
```

Comment:
if we're have a request body' it's more correct to use post and not get method

Reply:
fixed